gobblin-core-base/src/main/java/gobblin/writer/AsyncDataWriter.java

```diff
@@ -10,25 +10,33 @@
  * CONDITIONS OF ANY KIND, either express or implied.
  */
 
-package gobblin.kafka.writer;
+package gobblin.writer;
```

Comment:
Where would you recommend putting the AsyncWriterManager (a class that bridges the sync and async worlds).